lib/src/value/number.ts

```diff
@@ -616,7 +616,7 @@ export class SassNumber extends Value {
       return valueError(
         `Expected $this to have ${
           newNumerators.size + newDenominators.size > 1 ? 'units' : 'unit'
-        } ${unitString(newNumerators, newDenominators)}.`,
+        } ${unitString(newNumerators, newDenominators, true)}.`,
```

Comment:
Given this is the only place `unitString` is called with `verboseNoUnits = true`, I think it makes more sense to just eliminate the `verboseNoUnits` parameter and instead just only call `unitString` here if there are units, since right now this error's message would be `Expected $this to have unit no units`. It would be clearer to just error with `Expected $this to have no units` in that case.